RIPS/rip-7711.md

```diff
@@ -148,12 +148,11 @@ Using `SELFDESTRUCT` is only allowed within the same **frame** the contract was
 UNUSED_GAS_PENALTY = 10
 ```
 
-A penalty of `UNUSED_GAS_PENALTY` percent of the entire unused gas limit is charged from the
+A penalty of `UNUSED_GAS_PENALTY` percent of the entire unused "execution gas limit" is charged from the
```

Comment:
Are there any specific reasons to impose unused gas penalty only to execution gas limit? I was wondering if there's an attack that fills validation gas limit a lot and disturbs a block builder from building blocks efficiently.

Reply:
We discussed this briefly and it seems like the block builder has to run all the validation frames of RIP-7711 transactions in a block anyways.
Therefore, it will know precisely how much gas each validation frame consumes by the time it has a "valid block candidate".
The block builder can only start running execution frames after it has this "candidate", meaning the validation frame's consumption should be known. It's the execution frame's gas consumption is still unknown at this point.
Does this make sense or did we miss something?

Reply:
Um that sounds pretty reasonable! Thanks for the answer, I'll resolve this one.